src/network/api/homePageApi.ts

```diff
@@ -0,0 +1,61 @@
+import NetworkCall from "../networkCall";
+import { message } from "antd";
+import { store } from "../../store";
+import HomePageRequests from "../request/homePageRequests";
+import {
+  setCanonizedNameSpaces,
+  setCanonizedTopics,
+  pushToCanonizedTopics,
+} from "../../store/slices/homePageSlice";
+
+export const getCanonizedTopicsApi = async (reqBody, loadMore = false) => {
+  try {
+    const topics = await NetworkCall.fetch(
+      HomePageRequests.getCanonizedTopics(reqBody),
+      false
+    );
+    if (loadMore) {
+      store.dispatch(pushToCanonizedTopics(topics?.data));
+    } else {
+      store.dispatch(setCanonizedTopics(topics?.data));
+    }
+    return topics?.data;
+  } catch (error) {
+    // message.error(error.message);
+  }
+};
+
+export const getCanonizedNameSpacesApi = async () => {
+  try {
+    const nameSpaces = await NetworkCall.fetch(
```

Comment:
Return the 

```javascript
    const nameSpaces = await NetworkCall.fetch(
      HomePageRequests.getCanonizedNameSpaces()
    );

    return nameSpaces;
``` 
with

```javascript
    return await NetworkCall.fetch(
      HomePageRequests.getCanonizedNameSpaces()
    );

```

We don't need a variable

Reply:
Fixed